README.adoc

```diff
@@ -393,6 +393,33 @@ layrry-launcher-{layrry-version}-all.jar \
   --properties https://server:port/path/to/versions.properties
 ----
 
+=== Proxy Configuration
+
+You may need to configure a proxy when using the remote configuration feature. The following properties may be used to
```

Comment:
When you say "properties", what does that mean -- system properties, or entries in the Layrry properties file? Worth clarifying.

Reply:
Both actually. Commit 15bb4dc solves the problem of downloading layers & properties URLs when a proxy is needed. In this case properties must come from `System` properties because the external properties URL may not be yet available. I'll be sure to add more info to the README.